THIRD-PARTY-LICENSES.txt

```diff
@@ -0,0 +1,386 @@
+     (Apache 2.0) error-prone annotations (com.google.errorprone:error_prone_annotations:2.2.0 - http://nexus.sonatype.org/oss-repository-hosting.html/error_prone_parent/error_prone_annotations)
+     (The Apache Software License, Version 2.0) Guava InternalFutureFailureAccess and InternalFutures (com.google.guava:failureaccess:1.0.1 - https://github.com/google/guava/failureaccess)
+     (The Apache Software License, Version 2.0) Guava: Google Core Libraries for Java (com.google.guava:guava:27.0.1-jre - https://github.com/google/guava/guava)
+     (The Apache Software License, Version 2.0) Guava ListenableFuture only (com.google.guava:listenablefuture:9999.0-empty-to-avoid-conflict-with-guava - https://github.com/google/guava/listenablefuture)
+     (The Apache Software License, Version 2.0) Google HTTP Client Library for Java (com.google.http-client:google-http-client:1.28.0 - https://github.com/googleapis/google-http-java-client/google-http-client)
+     (The Apache Software License, Version 2.0) Apache HTTP transport for the Google HTTP Client Library for Java. (com.google.http-client:google-http-client-apache:2.0.0 - https://github.com/googleapis/google-http-java-client/google-http-client-apache)
+     (The Apache Software License, Version 2.0) Jackson extensions to the Google HTTP Client Library for Java. (com.google.http-client:google-http-client-jackson:1.28.0 - https://github.com/googleapis/google-http-java-client/google-http-client-jackson)
+     (The Apache Software License, Version 2.0) Jackson 2 extensions to the Google HTTP Client Library for Java. (com.google.http-client:google-http-client-jackson2:1.28.0 - https://github.com/googleapis/google-http-java-client/google-http-client-jackson2)
+     (The Apache Software License, Version 2.0) J2ObjC Annotations (com.google.j2objc:j2objc-annotations:1.1 - https://github.com/google/j2objc/)
+     (The Apache Software License, Version 2.0) Google OAuth Client Library for Java (com.google.oauth-client:google-oauth-client:1.28.0 - https://github.com/googleapis/google-oauth-java-client/google-oauth-client)
+     (The Apache Software License, Version 2.0) Java 6 (and higher) extensions to the Google OAuth Client Library for Java. (com.google.oauth-client:google-oauth-client-java6:1.28.0 - https://github.com/googleapis/google-oauth-java-client/google-oauth-client-java6)
+     (The Apache Software License, Version 2.0) JSON.simple (com.googlecode.json-simple:json-simple:1.1.1 - http://code.google.com/p/json-simple/)
+     (MIT) ficus (com.iheart:ficus_2.12:1.4.4 - http://iheartradio.github.io/ficus)
+     (MIT License) liquibase-slf4j (com.mattbertolini:liquibase-slf4j:2.0.0 - https://github.com/mattbertolini/liquibase-slf4j)
+     (The Apache License, Version 2.0) com.papertrail:profiler (com.papertrail:profiler:1.0.2 - https://github.com/papertrail/profiler)
+     (Unknown license) metrics-statsd-common (com.readytalk:metrics-statsd-common:4.2.0 - no url defined)
```

Comment:
Investigate

Reply:
I can now add license information for artifacts previously labeled 'Unknown license' but somehow the original labels 'Unknown license' and 'no url defined' don't go away. E.g:

(The Apache Software License, Version 2.0) (Unknown license) cwlavro-generated (io.cwl:cwlavro-generated:2.0.4.3 - no url defined)

Reply:
According to documentation https://www.mojohaus.org/license-maven-plugin/examples/example-thirdparty.html that is expected result